CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Deal with status code 204, when we see 204 in error path, we will not return data (from data layer only)
```

Comment:
It might be good to release this soon as it blocks some users

Reply:
Updated